bridge/bridge.go

```diff
@@ -68,6 +68,10 @@ func (b *Bridge) joinChannels(channels map[string]config.ChannelInfo, exists map
 	return nil
 }
 
+func (b *Bridge) ConfigFileUsed() string {
```

Comment:
Why is this function needed ? You have the Config.ConfigFileUsed() function already